park-dashboard/src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import { ThemeProvider } from "@material-ui/core/styles";
+import { createTheme } from '@material-ui/core/styles';
+import ParkIcon from '@mui/icons-material/Park';
+const useStyles = makeStyles((theme) => ({
```

Comment:
Nice way of adding CSS properties in the components. I should start doing the same for my own component as well.